### PR TITLE
add: adds reusable workflow container-build-push-feed

### PR DESCRIPTION
**Pull request by Balou9**
## What

adds reusable workflow container-build-push-feed

## References

https://jira.greenbone.net/browse/DEVOPS-1138

### CODE REVIEW

File: .github/workflows/container-build-push-2nd-gen.yml [hunk 1]
```diff
@@ -91,11 +100,25 @@ jobs:
...
+        id: set-data-container-tags
+        uses: docker-metadata-action@v5
+        if: ${{ inputs.data-container }} == "true"
+        with:
+          images: |
+            name=${{ inputs.image-url }}
+          tags: |
+            type=raw,value=community
+            type=raw,value=latest
+            type=raw,value=${{ inputs.feed-info }}
```
Comment (y0urself) [inline .github/workflows/container-build-push-2nd-gen.yml:120]:
```suggestion
          tags: $${ inputs.image-tags } 
```

File: .github/workflows/container-build-push-2nd-gen.yml [hunk 2]
```diff
@@ -109,15 +132,16 @@ jobs:
...
             type=semver,pattern={{major}}.{{minor}}
             type=semver,pattern={{major}}
             type=edge
-            type=ref,event=pr
+            type=raw,value=${{ github.ref_name }},,enable=${{ github.event_name == 'pull_request' }}
             type=raw,value=latest,enable=${{ steps.latest.outputs.is-latest-tag == 'true' }}
             type=raw,value=stable,enable=${{ steps.latest.outputs.is-latest-tag == 'true' }}
             type=raw,value=edge,enable=${{ inputs.ref-name == 'main' }}
-          image-platforms: ${{ matrix.platform }}
+            image-platforms: ${{ matrix.platform }}
```
Comment (y0urself) [inline .github/workflows/container-build-push-2nd-gen.yml:144]:
```suggestion
          image-platforms: ${{ matrix.platform }}
```
Why?

Comment (y0urself) [inline .github/workflows/container-build-push-2nd-gen.yml:144]:
```suggestion
              image-platforms: linux/amd64
```

File: .github/workflows/container-build-push-2nd-gen.yml [hunk 3]
```diff
@@ -184,7 +210,7 @@ jobs:
...
```
Comment (y0urself) [inline .github/workflows/container-build-push-2nd-gen.yml:213]:
```suggestion
              image-platforms: linux/arm64
```

**Review by pascalholthaus — APPROVED** [review]

**Pull request merged**